ecosystem/typescript/sdk/src/indexer/generated/queries.ts

```diff
@@ -45,6 +45,13 @@ export const GetAccountCurrentTokens = `
 }
     ${TokenDataFieldsFragmentDoc}
 ${CollectionDataFieldsFragmentDoc}`;
+export const GetIndexerLedgerInfo = `
+    query getIndexerLedgerInfo {
+  ledger_infos {
+    chain_id
+  }
+}
```

Comment:
Does this just get the latest ledger info or query historical data?

Reply:
just ledger info - which only has chain_id for now
https://cloud.hasura.io/public/graphiql?endpoint=https%3A%2F%2Findexer-devnet.staging.gcp.aptosdev.com%2Fv1%2Fgraphql